### ISSUE TITLE
[BUG] PDF generation fails with blank error

### ISSUE THREAD

**Issue by mithundas79** [alixaxel/chrome-aws-lambda#160]
## Environment
* `chrome-aws-lambda` Version: 5.2.1
* `puppeteer` / `puppeteer-core` Version: 5.2.1
* OS:  Linux
* Node.js Version:  13.x
* Lambda / GCF Runtime:  nodejs12.x

## Expected Behavior

Should create pdf

## Current Behavior

I get error 

```
{  error: {} }
```

**Comment by baharev** [comment]
@mithundas79 If the problematic line is indeed what you are saying (`browser = await chromium.puppeteer.launch({...`) then most of the code you are showing us is likely to be irrelevant. My guess would be that you are having a **configuration error / missing dependency**, and you cannot launch the browser. That's all.

If that is not the case, please reduce your example because it has too many moving parts.

Hard-code the HTML as a plain string in the JavaScript code. Get rid of everything specific to the Lambda and write the PDF directly to disc by passing a `path: '/tmp/myexample.pdf',` to the `await page.pdf({...})` call. We must be able to test everything locally and without `sam`.

If that still doesn't reveal where the problem is, try eliminating as much of the HTML and the JavaScript code as possible.

Long story short: **Please try to make your example minimal.**
